src/common/users.ts

```diff
@@ -320,24 +320,29 @@ export const getUserReadingTags = (
           COUNT(DISTINCT f.day) AS "readingDays"
         FROM
           filtered_view f
-          JOIN post_keyword pk ON f."postId" = pk."postId"
+          JOIN post_keyword pk ON f. "postId" = pk. "postId"
         WHERE
           pk.status = 'allow'
           AND pk.keyword != 'general-programming'
         GROUP BY
           pk.keyword
       )
       SELECT
-        tr.tag,
-        tr."readingDays",
+        tag,
+        "readingDays",
         tr."readingDays" * 1.0 / dd.total_days AS percentage,
         dd.total_days AS total
-      FROM
-        tag_readings tr
+      FROM (
+        SELECT
+          tr.tag,
+          tr. "readingDays"
```

Comment:
Nested select faster?

Reply:
According to result and everSQL yeah 👍
![Screenshot 2024-10-15 at 13 58 06](https://github.com/user-attachments/assets/39414252-ecd4-4bb1-8c33-d84afb0b8293)